tests/e2e/specs/in-game/one-offs/3_threes.spec.js

```diff
@@ -8,7 +9,6 @@ describe('Playing THREEs', () => {
   });
 
   it('Plays 3s with no cards in scrap', () => {
-    cy.skipOnGameStateApi();
```

Comment:
The only reason this wasn't in scope as of #1038 was the error message was inconsistent across legacy/gamestate api version of this move. Now that they all use the same i18n key, we can run this test in gamestate mode. Notably it doesn't require resolving the one-off or _doing_ it's effect, just requesting to make the move and validating that it's illegal because the scrap is empty